workflows/environment_upsert.go

```diff
@@ -273,6 +273,18 @@ func (workflow *environmentWorkflow) environmentElbUpserter(namespace string, en
 
 		stackParams["ElbInternal"] = strconv.FormatBool(environment.Loadbalancer.Internal)
 
+		if environment.Loadbalancer.AccessLogs.S3BucketName == "" {
+			stackParams["ElbAccessLogsS3BucketName"] = fmt.Sprintf("%s-loadbalancer-%s-access-logs", namespace, environment.Name)
```

Comment:
my preference is to leave defaults to be defined in the CFN template and the code not add a `stackParams` entry if the intention is to use the default value.  additionally, the default bucket name ought to include the account number and region name to avoid conflicts.

Reply:
Done.